web/pageObjects/landing_page.ts

```diff
@@ -2,7 +2,16 @@ import * as defaultPage from "./default_page"
 
 const btnSignInWithEmail = '//main//button[text()="Sign in with email"]'
 
-export async function clickSingInWithEmailBtn(): Promise<void> {
+//----------Actions----------//
+
+export async function clickBtnSingInWithEmail(): Promise<void> {
     await defaultPage.clickByLocator(btnSignInWithEmail)
 
+}
+
+//----------Waits----------//
+
+export async function waitUntilBtnSingInWithEmail(): Promise<void> {
```

Comment:
unify function names 
waitUntilBtnSortQuizesIsVisible() vs waitUntilBtnSingInWithEmail() 
need to use `isVisible` everywhere or do not use it at all

Reply:
Good point, nesupdantu pats kodel nepridejau `isVisible`